Installing-Oppia-using-Docker.md

```diff
@@ -107,11 +110,21 @@ For systems with 8GB RAM:
 
    > NOTE: This build is not related to production build in any way. This command runs `docker compose build` under the hood, which is used to build the images for services defined in the `docker-compose.yml` file. For more information, refer to the [official documentation](https://docs.docker.com/compose/reference/build/).
 
-### Launching the development server
+**Known issues and solutions**
```

Comment:
Maybe better to point them to the relevant section in Troubleshooting.md (https://github.com/oppia/oppia/wiki/Troubleshooting#docker-setup) and compile the troubleshooting instructions there?